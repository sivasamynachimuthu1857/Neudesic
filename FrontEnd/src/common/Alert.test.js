import React from "react";
import { render } from "@testing-library/react";
import Alert from "./Alert";

it("renders without crashing", function () {
  render(<Alert />);
});

it("matches snapshot for danger", function () {
  let message = ["OMG, everything is broken", "Everything needs to be redone"];
  const { asFragment } = render(<Alert type="danger" messages={message} />);
  expect(asFragment()).toMatchSnapshot();
});

it("matches snapshot for success", function () {
  let messages = ["Everything works great!"];
  const { asFragment } = render(<Alert type="success" messages={messages} />);
  expect(asFragment()).toMatchSnapshot();
});
