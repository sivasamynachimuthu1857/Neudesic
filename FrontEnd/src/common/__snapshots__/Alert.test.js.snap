// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot for danger 1`] = `
<DocumentFragment>
  <div
    class="alert alert-danger"
    role="alert"
  >
    <p
      class="mb-0 small"
    >
      OMG, everything is broken
    </p>
    <p
      class="mb-0 small"
    >
      Everything needs to be redone
    </p>
  </div>
</DocumentFragment>
`;

exports[`matches snapshot for success 1`] = `
<DocumentFragment>
  <div
    class="alert alert-success"
    role="alert"
  >
    <p
      class="mb-0 small"
    >
      Everything works great!
    </p>
  </div>
</DocumentFragment>
`;
